manifests/globals.pp

```diff
@@ -6,8 +6,10 @@
   $contrib_package_name = undef,
   $devel_package_name   = undef,
   $java_package_name    = undef,
+  $docs_package_name    = undef,
   $perl_package_name    = undef,
   $plperl_package_name  = undef,
+  $plpython_package_name= undef,
```

Comment:
i removed the spaces as per a previous review request to maintain the alignment. I could put the spaces back and/or rename 'plpython' to say 'plpythn' to maintain the alignment, which would you prefer?